my_2d_robot/src/calibration_2d.cpp

```diff
@@ -0,0 +1,130 @@
+#include "ceres/ceres.h"
+#include "glog/logging.h"
+#include "yaml-cpp/yaml.h"
+#include "ros/ros.h"
+#include "std_msgs/String.h"
+#include <sensor_msgs/JointState.h>
+#include <vector>
+
+using ceres::AutoDiffCostFunction;
+using ceres::CostFunction;
+using ceres::Problem;
+using ceres::Solve;
+using ceres::Solver;
+
+struct OffsetCalibration
+{
+public:
+  OffsetCalibration(double offset_joint1, double offset_joint2, double true_x, double true_y)
+    : true_x_(true_x), true_y_(true_y)
+  {
+    // Link lengths of robot (specified in launch file).
+    ros::NodeHandle n;
+    n.getParam("link_1", link1_);
+    n.getParam("link_2", link2_);
+
+    // Joint angles including angle offsets.
+    offset_joint1_ = offset_joint1;
```

Comment:
Since you are using initializer list syntax for  `: true_x_(true_x), true_y_(true_y)` above, you should continue this trend for `offset_joint1_` and `offset_joint2_`

Reference: https://en.cppreference.com/w/cpp/language/constructor